Automattic/wp-calypso
# Recurring Payments: a little onboarding + atomic fix

I realized that 2 groups may stumble upon the new earn tab:
- a) users without context
- b) atomic users

This PR disables feature for Jetpack older than 7.4 (and 7.4 is enabling it). Additionally it shows a little onboarding to explain what is it about.

### Screenshots

![Zrzut ekranu 2019-06-1 o 18 58 05](https://user-images.githubusercontent.com/3775068/58751523-d5043500-849f-11e9-9e43-17c5f12538af.png)
![Zrzut ekranu 2019-06-1 o 19 12 17](https://user-images.githubusercontent.com/3775068/58751639-3d9fe180-84a1-11e9-9383-7c862c16134b.png)
![Zrzut ekranu 2019-06-1 o 19 11 54](https://user-images.githubusercontent.com/3775068/58751640-3f69a500-84a1-11e9-9838-1e8cb4400e49.png)



### Testing instructions

1. Open in atomic. You should get "Upgrade to jetpack 7.4" Not ideal, but works
2. Open in a site you did not connect. You should get little onboarding.
